tiatoolbox/models/architecture/vanilla.py

```diff
@@ -268,3 +321,111 @@ def infer_batch(
             output = model(img_patches_device)
         # Output should be a single tensor or scalar
         return [output.cpu().numpy()]
+
+
+class TimmModel(CNNModel):
```

Comment:
```suggestion
class TimmModel(ModelABC):
```

As you are using ModelABC.__init__, you do not need CNNModel unless there is any other functionality you need.

Reply:
There are the `infer_batch()` and `_postproc()` methods, which `TimmModel` inherits from `CNNModel`.